zircon.jvm.examples/src/main/kotlin/org/hexworks/zircon/examples/fragments/table/Gender.kt

```diff
@@ -0,0 +1,6 @@
+package org.hexworks.zircon.examples.fragments.table
+
+enum class Gender {
```

Comment:
This might seem silly, but do you mind if we drop this particular column from the example? Seems needlessly complicating :)

If you just need an enum example, how about, um...`Height { SHORT, AVERAGE, TALL }`.